src/MaterialDesignControls/Implementations/CustomMaterial3/ContentBox.cs

```diff
@@ -0,0 +1,55 @@
+﻿using Xamarin.Forms;
+
+namespace Plugin.MaterialDesignControls.Material3.Implementations
+{
+    public partial class ContentBox : Frame
+    {
+        #region Attributes & Properties
+
+        public static readonly BindableProperty BorderWidthProperty =
+            BindableProperty.Create(nameof(BorderWidth), typeof(float), typeof(ContentBox), 0f);
+
+        public float BorderWidth
+        {
+            get { return (float)GetValue(BorderWidthProperty); }
+            set { SetValue(BorderWidthProperty, value); }
+        }
+
+        public static readonly BindableProperty CornerRadiusTopLeftProperty =
```

Comment:
I'm thinking if we can use a only one bindable property to the CornerRadius which will override the base one and has the same type than the Margin and Padding properties, where you can do:
- "10" (apply 10 to all corners).
- "10,0" (apply 10 to top corners and 0 to bottom corners).
- "10,0,0,0" (apply 10 only to top-left corner).